lib/opentok/token_generator.rb

```diff
@@ -81,6 +81,15 @@ def generate_token
         end
         data_params[:connection_data] = data
       end
+
+      if opts[:initial_layout_classes]
```

Comment:
Is https://tokbox.com/developer/guides/archiving/layout-control.html the correct docs for this layout param? I see a param called `layoutClassList`, so which one is correct? For `layoutClassList`, it indicated that it's an array of string. The docs does not seem to be clear me.